CONTRIBUTING.md

```diff
@@ -1,152 +1,196 @@
+- An explanation of your use case
 
-**For Bugs:**
+### Report bugs
+
+If you have found a bug in the Light Client or the JS SDK, then please use the **Bug** issue template and provide the following infos:
 
 - A short description of the problem.
-- Detailed description of your system, SDK version, environment (e.g. Metamask), Wallet version if you are using the wallet.
-- What was the exact unexpected thing that occured.
-- What you were expecting to happen instead.
+- The steps you need take in order to reproduce the behavior
+- What was the expected result
+- What was the actual result
 
-### Creating a Pull Request
+## Creating a pull request
 
-If you have some coding abilities and would like to contribute to the actual
-codebase of Raiden then you can open a Pull Request(PR) against the repository.
+If you have coding experience and would like to contribute to codebase, then you can open a pull request against the repository.
 
 If you are interested in contributing make sure that there is an issue about it. Express interest, by picking the issue so that core developers know that you are working on the issue.
 
-All PRs should be:
+All pull requests should be:
 
 - Self-contained.
 - As short as possible and address a single issue or even a part of an issue.
-  Consider breaking long PRs into smaller ones.
+  Consider breaking long pull requests into smaller ones.
 
-In order for a Pull Request to get merged into the main repository you should
-have one approved review from one of the core developers of Raiden and also all
-Continuous Integration tests should be passing and the CI build should be
-green.
-
-Additionally you need to sign the raiden project CLA (Contributor License
-Agreement). Our CLA bot will help you with that after you created a pull
-request. If you or your employer do not hold the whole copyright of the
-authorship submitted we can not accept your contribution.
+### Installation
 
-For frequent contributors with write access to the repository we have a set of rules on Pull Requests to signal to our colleagues what the current state of the PR is. These are:
+These are the required external dependencies for development:
 
-- A ready for review Pull Request means that you consider your work done and it is currently ready for a reviewer to have a look at.
-- A draft Pull Request is considered work in progress and it is not yet ready for review.
+- Node >=10.13.0
+- A Web3 enabled browser (e.g. through [MetaMask](https://metamask.io))
+- git
 
-### Pull Request Reviews
+Start by getting the source code:
 
-It is the responsibility of the author to ask for at least one person to review their Pull Request. That person should know the area of the code being changed. If the chosen reviewer does not feel confident in the review, they can then ask for someone else to additionally look at the code.
+```bash
+$ git clone --recurse-submodules https://github.com/raiden-network/light-client.git
```

Comment:
I'm guessing that `--recurse-submodules` automatically clones the contracts repo? I've always done it manually. This is awesome. Perhaps I should start readin the docs more carefully. :wink:

Reply:
Yup, exactly!